CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 - added the current music track and timestamp to the savegame so they now persist on load (#419)
 - added the triggered music tracks to the savegame so one shot tracks don't replay on load (#371)
 - added forward/backward input detection in line with TR2+ for jump-twists (#931)
+- added an option to restore the mummy in City of Khamoon room 25, similar to the PS version (#886)
```

Comment:
Idc personally but all other changelog entries use `PS1`. Maybe we should be consistent? Or we could retroactively change them all to `PS` or `PSX`. Whatever @rr- prefers.

Reply:
I'm fine with both, props for consistency

Reply:
Changed to PS1 to match.